src/cbmc/bmc.h

```diff
@@ -294,6 +296,8 @@ class path_explorert:public bmct
   " --unwind nr                  unwind nr times\n" \
   " --unwindset L:B,...          unwind loop L with a bound of B\n" \
   "                              (use --show-loops to get the loop IDs)\n"  \
+  " --partial-merge              do path exploration only after executing\n" \
+  "                              __CPROVER_begin_path_explore()\n"  \
```

Comment:
Perhaps "encountering __CPROVER_begin_path_explore in the input program" would be clearer